tir/technologies/webapp_internal.py

```diff
@@ -1037,6 +1074,12 @@ def filling_environment(self, shadow_root=None, container=None):
                     if click_type > 3:
                         click_type = 1
 
+        if env_value.strip() != self.config.module.strip() and enable:
+            message = "Cant_fill_program"
+            self.log.take_screenshot_log(self.driver, description=message, stack_item=self.log.get_testcase_stack())
+            self.log_error(message)
+            raise ValueError(message)
```

Comment:
We need understand what the intentions for this change. We understood that you are tryng to test the access screens. If that's the intention, it is not TIR purpose.